azplugins/flow.py

```diff
@@ -757,3 +776,15 @@ def velocity(self):
             return np.divide(self._velocity, self._counts, out=np.zeros(self.bins), where=self._counts > 0)/self.samples
         else:
             return np.zeros(self.bins)
+
+    @property
+    def temperature(self):
```

Comment:
For consistency with the rest of HOOMD, let's call this property `kT` throughout because it is technically the thermal energy (Boltzmann's constant nominally being 1), even though this is essentially temperature.